syntax = "proto2";

package fyTcuSystemInfo; //TCU系统信息

message TcuSystemMenu
{
	required int32  actual_clutch_position = 1;		//离合实际位置, 0
    	required uint32 tcu_fault_code = 2;			//TCU故障信息, 00000000（2进制显示）
    	required int32  axis_rpm = 3;				//变速箱输出轴转速, 0 RPM
    	required int32  fmi = 4;				//FMI值, 0
    	required int32  cm = 5;					//CM值, 0
    	required uint32 oc = 6;			        	//OC, 0（16进制显示）
}
